assets/js/phoenix_live_view.js

```diff
@@ -1453,10 +1462,10 @@ class DOMPatch {
             DOM.mergeFocusedInput(fromEl, toEl)
             DOM.syncAttrsToProps(fromEl)
             updates.push(fromEl)
-            return false
+            return false  
           } else {
-            if(DOM.isPhxUpdate(toEl, phxUpdate, ["append", "prepend"])){
-                appendPrependUpdates.push(new DOMAppendPrependUpdate(fromEl, toEl, toEl.getAttribute(phxUpdate)))
+            if(DOM.isPhxUpdate(toEl, this.binding(PHX_UPDATE), ["append", "prepend"])){
+                appendPrependUpdates.push(new DOMAppendPrependUpdate(fromEl, toEl, this.binding.bind(this)))
```

Comment:
`this.binding.bind(this)` is kind of ugly.

I was thinking about passing the livesocket into the `DOMAppendPrependUpdate` class instead of the `binding` function. What do you think?